model-api/xml-model/src/main/java/org/camunda/bpm/model/xml/impl/validation/ModelValidationResultsImpl.java

```diff
@@ -44,6 +45,15 @@ public ModelValidationResultsImpl(
     this.warningCount = warningCount;
   }
 
+  public ModelValidationResultsImpl(ValidationResults... validationResults) {
+    collectedResults = new HashMap<>();
+    for (var entry : validationResults) {
+      collectedResults.putAll(entry.getResults());
+      errorCount += entry.getErrorCount();
+      warningCount += entry.getWarinigCount();
+    }
+  }
```

Comment:
🔧 You could explain why such a constructor is useful to consumers of this library, in the commit message.

Reply:
done